src/index.ts

```diff
@@ -4,19 +4,34 @@ import { DocsParser } from './DocsParser';
 
 type ParseOptions = {
   baseDirectory: string;
+  useReadme: boolean;
   moduleVersion: string;
   packageMode?: 'single' | 'multi';
 };
 
 export async function parseDocs(options: ParseOptions) {
   const packageMode = options.packageMode || 'single';
-  const electronDocsPath = path.resolve(options.baseDirectory, 'docs', 'api');
+
+  const apiDocsPath = options.baseDirectory || path.resolve('./', 'docs', 'api');
+  const structuresPath = path.resolve(apiDocsPath, 'structures');
+
+  let structures: string[] = [];
+  let apis: string[] = [];
+
+  if (options.useReadme) {
+    apis = [path.resolve(options.baseDirectory, 'README.md')];
```

Comment:
We should assert that that file actually exists, the methods these paths are passed to don't do any validity checking.  They assume the path being passed in is legit.